### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ananyo2012**
* Update Conference info
* Add Getting Started Guide
* Add Code of Conduct

### CODE REVIEW

**Review by sayanchowdhury — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,14 +2,29 @@
...
 - Dev Sprints: October 4-5
 
-**Venue**
-Sheraton Grand Convention Center, Whitefield, Bengaluru
+## Getting Started
+
+This site is built with [Jekyll](https://jekyllrb.com/).
+
+* Install Jekyll. Follow the first 2 steps in https://jekyllrb.com/docs/.
+* Run `git clone https://github.com/pythonindia/inpycon2020`.
```
Comment (Sc0rpi0n101) [inline README.md]: I think telling the contributors to clone this repo instead of their fork might not be a good idea.

Comment (Sc0rpi0n101) [reply README.md]:
```suggestion
* Fork the repository and clone to your machine by using `git clone https://github.com/<username>/inpycon2020`.
```

File: README.md [hunk 2]
```diff
@@ -2,14 +2,29 @@
...
+* `cd` to the repo and run `bundle install` to install all dependencies.
+* Run `bundle exec jekyll serve`.
+* Visit `http://localhost:4000/2020`.
```
Comment (Sc0rpi0n101) [inline README.md]:
```suggestion
* Visit the local development server at `http://localhost:4000/2020`.
```

**Review by Sc0rpi0n101 — COMMENTED** [review]
Here are some links that might be useful to new contributors if you want to add them in the readme 

https://guides.github.com/activities/forking/
https://guides.github.com/introduction/git-handbook/
https://guides.github.com/introduction/flow/
https://chris.beams.io/posts/git-commit/
https://git-scm.com/book/en/v2

**Pull request merged**